tests/suites/casper/request.coffee

```diff
@@ -5,6 +7,10 @@ utils = require "utils"
 if utils.ltVersion(phantom.version, '1.9.0')
   casper.test.skip(6, 'PhantomJS version <1.9.0 does not implement request.abort()')
   casper.test.done()
+# FIXME: abort test does not work for slimerjs, either -- disable
+else if true
```

Comment:
Yes -- fully aware of that... :-)
- phantomjs 1.9 and below could not pass (no support for abort)
- phantomjs 2.x could not pass (no support for coffee)
- slimerjs obviously never passed
- but there is a slight chance that slimerjs *might* pass after tweaking, so left it in with FIXME instead of removing the test altogether